apps/condo/pages/ticket/index.tsx

```diff
@@ -195,6 +198,17 @@ export const TicketsPageContent = ({
                                                                         {WarrantiesLabel}
                                                                     </Checkbox>
                                                                 </Col>
+                                                                <Col>
+                                                                    <Checkbox
+                                                                        onChange={handleReturnedChange}
+                                                                        checked={returned}
+                                                                        style={CHECKBOX_STYLE}
+                                                                        eventName={'TicketFilterCheckboxWarranty'}
+                                                                        data-cy={'ticket__filter-isWarranty'}
```

Comment:
You should use unique string id for cypress attributes. This may break the test!